nikanorov-alex/Pr4/ContentSinoptik.py

```diff
@@ -0,0 +1,45 @@
+from Link import Link
+from WeatherFather import WeatherFather
+class ContentSinoptik(WeatherFather):
+    def __init__(self, html, mainClass, mainNextDay, tabsContentInner, temperature, pClassOne, pClassTwo, pClassThree, min, max):
+        super(ContentSinoptik, self).__init__(html, mainClass, mainNextDay, tabsContentInner, temperature, pClassOne, pClassTwo, pClassThree, min, max)
+
+    def outputInf(self, weatherList):
+        location_name = Link().enter_the_inf()
+        weatherFather = weatherList[0]
+        weatherFather2 = weatherList[1]
+        weatherFather3 = weatherList[2]
+
+        for i in weatherFather:
+            w1 = i
+        for i in weatherFather2:
+            w2 = i
+        for i in weatherFather3:
+            w3 = i
+
+        
+        print()
+        print("------------------------------------------------------")
+        print("\t\t\t\t\t\t\t\t\t\tПогода з сайту Sinoptik " + location_name)
+        print("День:" + w1['title'])
+        print("Число: " + w1['date1'], ' ', w1['date2'])
+        print("Температура: :" + w1['temp'], " ||  " + w1['temp2'])
+        print()
```

Comment:
You can try making it more general. E.g. it could be a *UI* class that shows the table. But the source (the site name) and the corresponding date could be passed through parameters.